cypress/support/fragments/settings/users/limits.js

```diff
@@ -44,4 +45,14 @@ export default {
       saveButton.click(),
     ]);
   },
+  setLimit: (limitName, number) => {
+    Conditions.getConditionsViaApi().then(conditions => {
+      return conditions.find(condition => condition.name === limitName).id;
+    }).then((limitId) => {
+      cy.do([
+        TextField({ name: limitId }).fillIn(number),
+        saveButton.click(),
+      ]);
+    });
+  }
```

Comment:
Can I delete other setMaximumNumber... methods and instead use this one in other tests where the limit needs to be specified? In my opinion, this method is better and more versatile.

Reply:
Sure! Great approach!